libraries/cli/include/cli/config_jsons/mainnet/mainnet_config.json

```diff
@@ -2,6 +2,7 @@
   "data_path": "",
   "is_light_node": false,
   "final_chain_cache_in_blocks": 5,
+  "report_malicious_behaviour" : true,
```

Comment:
do we want it enabled by default for all nodes in the network? I think it could be too much